CodeOfConduct.md

```diff
@@ -48,6 +48,11 @@ This section contains things which are less specific and often less problematic
 2. Players may leave games for a variety of valid reasons. Players who make a habit of leaving ongoing matches extremely early risk being met with moderation action. Matches thought to suffer from issues (balance, bugs etc.) require a restart and can be stopped by calling a stop vote with `!cv stop` in chat.
 3. Leaving a game very early because you don't like the teams is also not acceptable, instead call a stop vote `!cv stop`. Note that leaving mid-game is acceptable as you are unlikely to disadvantage your allies to such an extent. If abused persistently it will be moderated.
 
+### 5. Boss abuse
+1. Boss mode is a tool to facilitate a good experience for everyone. Using boss mode to abuse other players is not acceptable, and may result in removal of boss privileges.
+2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
```

Comment:
```suggestion
2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
    - if you are the boss of a lobby, you can still start a kickban *vote* on a player using `!cv kickban <nickname>` ("coordinator vote") instead of `!kickban <nickname>`.
```

Reply:
Small tweak: s/nickname/playername/g, since that terminology is already used elsewhere in the CoC:
```suggestion
2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
   - if you are the boss of a lobby, you can still start a kickban *vote* on a player using `!cv kickban <playername>` ("coordinator vote") instead of `!kickban <playername>`.
```